hw-7/task3.js

```diff
@@ -0,0 +1,19 @@
+function divide (numerator, denominator) {
+    if (denominator === 0) {
+        throw new Error("You can't devide by zero. Please select a valid value.")
+    } else if (typeof numerator !== "number" || typeof denominator !== "number"){
+        throw new Error("Your input should be a number. Please select another value.")
+    } else {
+        return numerator/denominator
+    }
```

Comment:
better use dedicated `if` for each validation rule and return numerator/denominator without `else`

it is recommendation